zrml/prediction-markets/src/lib.rs

```diff
@@ -279,6 +280,16 @@ mod pallet {
             Self::do_buy_complete_set(sender, market_id, amount)
         }
 
+        #[pallet::weight(T::WeightInfo::dispute(T::SimpleDisputes::max_disputes()))]
+        pub fn dispute(
+            origin: OriginFor<T>,
+            market_id: MarketIdOf<T>,
+            outcome: OutcomeReport,
+        ) -> DispatchResultWithPostInfo {
+            let [weight, max_weight] = T::SimpleDisputes::on_dispute(origin, market_id, outcome)?;
+            Self::calculate_actual_weight(&T::WeightInfo::dispute, weight, max_weight)
+        }
+
```

Comment:
@lsaether Re-adding this call until the actual court pallet is implemented

Reply:
I actually think we should keep this like this, then pattern match on the `DisputeMechanism` for the `market_id` and route the call to either `SimpleDisputes` or `Court` based on what's configured.

Reply:
Got it!